packages/cfd/src/Containers/cfd-password-manager-modal.tsx

```diff
@@ -355,7 +357,7 @@ const CFDPasswordManagerModal = observer(
                         title={getTitle()}
                         toggleModal={toggleModal}
                         height='688px'
-                        width='904px'
+                        width={!isDesktop ? '70rem' : '90.4rem'}
```

Comment:
i don't think we need this condition since this modal is wrapped in `DesktopWrapper`

Reply:
please replace `DesktopWrapper` and `MobileWrapper`

Reply:
DesktopWrapper can be removed but MobileWrapper needs to be kept as it is since we have some differences in Tablet and Mobile view. Take modal vs full page dialogue for password set as an instance.

Reply:
We should have had a Tablet wrapper for tablet view instead of useDevice that we're using for tablet view. If we want to have the same behavior for both mobile and Tablet we need to remove the MobileWrapper but since they are not exactly the same right now we cannot do so.

Reply:
please make sure to remove both `DesktopWrapper` and `MobileWrapper`. We only need to refer to the sizes from the ui library.

Reply:
I will do another round of checking on Tablet view after this PR is moved forward. As this is production issue We'd better get this merged ASAP and we focus on removal of DesktopWrapper and MobileWrapper after that.